src/open-id-connect-navigation-strategies.ts

```diff
@@ -42,7 +42,7 @@ export default class OpenIdConnectNavigationStrategies {
             // And we will receive one of two errors:
             // 'No matching state found in storage' or
             // 'No state in response'
-            const url: string = null;
+            const url: string | undefined = undefined;
```

Comment:
Note-to-self: Consider changing this to `this.userManager.signinSilentCallback(undefined)` instead of having an explanatory variable.